woocommerce/woocommerce
# Product Gallery: Update div elements to ul/li for better accessibility

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





Closes #43304 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to Appearance > Editor > Templates > Single Product (clear customizations)
2. Click on the editor and you should see "Transform into blocks" action button.
3. Add the Product Gallery block.
4. Open a Product that has multiple images in its gallery.
5. Ensure all of the Product Gallery inner blocks (Large Image, Thumbnails, Pager, Next/Previous Buttons etc.) are working correctly and there are no visual changes.
6. Check if the markup of images is utilizing `ul` and `li` instead of `div` (test using a browser inspector).

```
<ul class="wc-block-product-gallery-large-image__container">
    <li class="wc-block-product-gallery-large-image__wrapper"></li>
    <li class="wc-block-product-gallery-large-image__wrapper"></li>
    <li class="wc-block-product-gallery-large-image__wrapper"></li>
</ul>
```

instead of:

```
<div class="wc-block-product-gallery-large-image__container">
    <div class="wc-block-product-gallery-large-image__wrapper"></div>
    <div class="wc-block-product-gallery-large-image__wrapper"></div>
    <div class="wc-block-product-gallery-large-image__wrapper"></div>
</div>

```



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
